### DISCUSSION TITLE
How can you delete projects?

### DISCUSSION THREAD

**Discussion by skanda890** [community/community#75927] [Discussions] [answered]
### Select Topic Area

Question

### Body

How can you delete projects in web or mobile platform?

**Comment by ghost** [comment, marked as answer]
Hello @skanda890 

To delete your project in github: 

1. Navigate to your project.
2. In the top-right, click ... to open the menu (click on 3 dot icon).
3. in the menu, click  Settings to access the project settings.
4. scroll down and In the "Danger zone" section, next to "Delete project," click Delete this project.
5. type your project name, Click I understand the consequences, delete this project.

**However:**
you can close your project instead of deleting it. closing project allows you to re-open it if you change your mind in the future. 

you can find more information about deleting/closing github project [there](https://docs.github.com/en/issues/planning-and-tracking-with-projects/managing-your-project/closing-and-deleting-your-projects)

Hope this helps
